src/common/hooks/search/use-recent-items.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
-import { useRecoilState } from 'recoil';
-import { searchRecentlyViewedItemsState } from '@store/search';
+
+import { searchRecentlyViewedItemsState } from '@store/recoil/search';
```

Comment:
What does `recoil` mean here?

Reply:
this folder should be moved up, I'll do that after your initial review -- it was part of my refactor to move all recoil state to its own folder, but now it's using jotai